packages/hash/api/src/db/adapter.ts

```diff
@@ -562,58 +576,70 @@ export interface DbClient {
-    operation: object;
+    updatedOperation: object;
+    updatedByAccountId: string;
   }): Promise<DbAggregation>;
 
   /**
-   * Get an aggregation by its id.
+   * Get an aggregation by its aggregation id.
    *
-   * @param params.aggregationId the id of the aggregation
+   * @param {string} params.sourceAccountId - the account id of the source entity
+   * @param {string} params.aggregationId - the id of the aggregation
+   * @returns {DbAggregation | null} the aggregation with if found in the datastore, otherwise `null`
    */
   getAggregation(params: {
+    sourceAccountId: string;
     aggregationId: string;
   }): Promise<DbAggregation | null>;
 
   /**
    * Get an aggregation by its source entity and path.
    *
-   * @param params.sourceAccountId the account id of the source entity
-   * @param params.sourceEntityId the entity id of the source entity
-   * @param params.sourceEntityVersionId the entityVersionId of the source entity (optional)
-   * @param params.path the aggregation path
+   * @param {string} params.sourceAccountId - the account id of the source entity
+   * @param {string} params.sourceEntityId - the entity id of the source entity
+   * @param {string} params.path - the aggregation path
+   * @param {Date} [params.activeAt] - the timestamp at which the aggregation was active, when the source entity is versioned
+   * @returns {DbAggregation | null} the aggregation if found in the datastore, otherwise `null`
    */
   getEntityAggregationByPath(params: {
     sourceAccountId: string;
     sourceEntityId: string;
-    sourceEntityVersionId?: string;
     path: string;
+    activeAt?: Date;
   }): Promise<DbAggregation | null>;
 
   /**
    * Get all aggregations for an entity.
    *
-   * @param params.sourceAccountId the account id of the source entity
-   * @param params.sourceEntityId the entity id of the source entity
-   * @param params.sourceEntityVersionId the entityVersionId of the source entity (optional)
+   * @param {string} params.sourceAccountId - the account id of the source entity
+   * @param {string} params.sourceEntityId - the entity id of the source entity
+   * @param {Date} [params.activeAt] - the timestamp at which the aggregations were active, when the source entity is versioned
+   * @returns {DbAggregation[]} the aggregations of the entity
    */
   getEntityAggregations(params: {
     sourceAccountId: string;
     sourceEntityId: string;
-    sourceEntityVersionId?: string;
+    activeAt?: Date;
   }): Promise<DbAggregation[]>;
 
   /**
    * Delete an existing aggregation.
    *
-   * @param params.aggregationId the id of the aggregation
-   * @param params.deletedByAccountId the account id of the user that deleted the aggregation
+   * @param {string} params.sourceAccountId - the account id of the source entity
+   * @param {string} params.aggregationId - the id of the aggregation
+   * @param {string} params.deletedByAccountId - the account id of the user that deleted the aggregation
```

Comment:
Same comment as for `createdByAccountId` - as this field becomes something else in the db, we could note that in the doc, or update the param name to match

Reply:
https://github.com/hashintel/hash/pull/484#discussion_r849245485